pkg/apis/cdap/v1alpha1/constants.go

```diff
@@ -54,20 +54,14 @@ const (
 	confUserInterfaceBindPort = "dashboard.bind.port"
 
 	// Value for the local data directory
-	localDataDir              = "/data"
-	instanceLabel             = "cdap.instance"
-	containerLabel            = "cdap.container"
-	templateDir               = "templates/"
-	deploymentTemplate        = "cdap-deployment.yaml"
-	uiDeploymentTemplate      = "cdap-ui-deployment.yaml"
-	statefulSetTemplate       = "cdap-sts.yaml"
-	serviceTemplate           = "cdap-service.yaml"
```

Comment:
Why some constants are moved while some are not?

Reply:
Ideally local constants live where there are used (same package as).  
Global constants live in appropriate package and are exported so that they can be used by other packages.

Reply:
Would mind moving all the local constants to `cdapmaster_controller` (e.g. all the `defaultXXX` below) as well? Thanks.

Reply:
i checked. They are used only in ApplyDefault within the same package. It makes sense for applydefaults to be part of api/v1alpha1.